Initial Commit For api-04-remote-and-local Objective (#59)
TwilioQuest/twilioquest-api-academy: levels/lovelace_tower/objectives/api-04-remote-and-local/walkthrough.md

# What is an API?

> [APIs](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Client-side_web_APIs/Introduction#what_are_apis) (Application Programming Interface) provide ways in which we can interact with other computer programs. They do so by publicly exposing what are called `endpoints`. Each `endpoint` represents a different service, and made accessible via a unique URL (Uniform Resource Locator) and [method](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods).

Reviewer: ```suggestion
[APIs](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Client-side_web_APIs/Introduction#what_are_apis) (Application Programming Interface) provide a way to interact with other computer programs. They do so by publicly exposing `endpoints`, with each `endpoint` representing a different service, and made accessible via a unique URL (Uniform Resource Locator) and [method](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods).
```

## What is JSON?

> [JSON](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON) (Javascript Object Notation) is a common way of formatting data, and is often used when making requests to endpoints that provide information in their response. It's important to note that when using `fetch` and receiving JSON, we must [parse](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON#static_methods) (i.e. have the computer anylize) the JSON data before using it.

Reviewer: ```suggestion
[JSON](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON) (Javascript Object Notation) is a common format used to represent and exchange data, and is often used when making requests to endpoints that provide information in their response. It's important to note that when using `fetch` and receiving JSON, we must [parse](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON#static_methods) (i.e. have the computer anylize) the JSON data before using it.
```
